src/ESPiLight.cpp

```diff
@@ -435,23 +437,25 @@ int ESPiLight::stringToPulseTrain(const String &data, uint16_t *codes,
   int spulse = data.indexOf('p') + 2;
   if (scode > 0 && (unsigned)scode < data.length() && spulse > 0 &&
       (unsigned)spulse < data.length()) {
-    int nrpulses = 0;
-    start = spulse;
+    unsigned int nrpulses = 0;
+    start = (unsigned)spulse;
```

Comment:
Now you are not using `static_cast<>`. We should be consistent for the whole project.

Reply:
This is mainly because this function is full of plain casts. I just tried to adopt the style at this point.

Reply:
Lets decide for a style for the hole project.

Reply:
For simple scalar types a plain cast is fine and often more readable. For everything else I would consider `static_cast()`.